flavel_game_noah.py

```diff
@@ -36,16 +24,17 @@ def display_loc_interactors(self):
 			interactor.see_me()
 
 	def go_to(self):
+		print "\n",
 		if self.visits == 0:
-			print "You arrive %s %s for the first time." % (self.modifier, self.get_name())
+			print "You arrive %s %s for the first time." % (self.modifier, self.call_name.capitalize())
```

Comment:
Check out the new python formatting: https://pyformat.info I think it's usually easier to read